winforms/src/toga_winforms/widgets/webview.py

```diff
@@ -136,6 +135,10 @@ def winforms_initialization_completed(self, sender, args):
             settings.IsSwipeNavigationEnabled = False
             settings.IsZoomControlEnabled = True
 
+            self.native.CoreWebView2.NavigationStarting += (
+                self.winforms_navigation_starting
```

Comment:
This almost certainly needs a WeakrefCallable wrapper.

Reply:
You mean like the OnWebViewLoadHandler(Protocol) class?

Reply:
Yes - you can see the same handling in `NavigationCompleted` and `CoreWebView2InitializationCompleted` in the WebView widget.

Reply:
OK, added the WeakrefCallable wrapper